src/maintenance/coverage_crossref_updater.py

```diff
@@ -0,0 +1,44 @@
+"""Updates the table for the `coverage_crossref` endpoint."""
+
+# standard library
+import argparse
+import sys
+import time
+
+# first party
+from delphi.epidata.acquisition.covidcast.database import Database
+from delphi_utils import get_structured_logger
+from delphi.epidata.client.delphi_epidata import Epidata
+
+
+def main(database_impl=Database):
+  """Updates the table for the `coverage_crossref`."""
+
+  logger = get_structured_logger("coverage_crossref_updater")
+  start_time = time.time()
+  database = database_impl()
```

Comment:
you should remove this since the dependency injection doesnt actually get used elsewhere

```suggestion
def main():
  """Updates the table for the `coverage_crossref`."""

  logger = get_structured_logger("coverage_crossref_updater")
  start_time = time.time()
  database = Database()
```